src/main/java/frc/robot/commands/auto/complex/BackUpAndBalance.java

```diff
@@ -13,8 +13,8 @@ public BackUpAndBalance(String relativePosition, Drive drive, Arm arm) {
     int aprilTag = 7;
     addCommands(
         new Initialize(aprilTag, relativePosition, drive, arm),
-        new StraightDriveToPose(Units.inchesToMeters(170.0), 0.0, 0.0, drive),
-        new StraightDriveToPose(Units.inchesToMeters(-75.0), 0.0, 0.0, drive),
+        new StraightDriveToPose(Units.inchesToMeters(175.0), 0.0, 0.0, drive),
```

Comment:
Can all these constants be moved to a `FieldConstants` file to give them names and be able to reuse them?

Reply:
the goal is to replace these with different positions already in fieldconstants, but this requires us to have more reliable pose estimation first